uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/APIWrapper/DefaultLimitRemoverTest.php

```diff
@@ -21,43 +29,37 @@ public function setUpHeadless() {
   }
 
   public function setUp() {
-    $this->defaultLimitRemoverObj = new CRM_HRCore_APIWrapper_DefaultLimitRemover();
-
-    $this->testParameters['params'] = [
-      'action' => 'get',
-      'entity' => 'contact',
-      'id' => 1,
-    ];
+    $this->defaultLimitRemover = new DefaultLimitRemover();
```

Comment:
Since you're not resetting `testParameters` on this method anymore, you should not share this variable among all the tests anymore (that is, you should not use as a class property). The reason is that one test should be isolated and not affected by changes made by other tests. One test might expect limit to always start as 0, but it will fail if it runs after another one that will change limit to 10.

So what needs to be done here is:
- Remove the `testParameters` property, as I mentioned on the previous comment
- On each test, create the the a variable with all the values it needs